inc/admin/namespace.php

```diff
@@ -43,11 +51,8 @@ function get_keywords_html( $post_id, $limit = 10 ) : string {
 				'<style>
 					.compat-field-hm-aws-rekognition-labels p { margin: 6px 0; }
 				</style>
-				<p>%s</p>
-				<p class="error message">%s: %s</p>',
-				esc_html__( 'There was an error analysing the image.', 'hm-aws-rekognition' ),
-				esc_html( (string) $label_errors->get_error_code() ),
-				esc_html( $label_errors->get_error_message() ?? 'Unspecified error' )
+				<p>%s</p>',
+				esc_html__( 'There was an error analysing the image. Check the error log for details.', 'hm-aws-rekognition' )
```

Comment:
Should be en-us here. Also, think this is still too technical, suggest:

```suggestion
				esc_html__( 'There was an error analyzing the image. Please try again later.', 'hm-aws-rekognition' )
```

Reply:
There's no way to try again from the UI right, only CLI. I'll change it to just the basic error message.

Reply:
Well I mean, refresh the page 😉